core/mondoo-gcp-inventory.mql.yaml

```diff
@@ -0,0 +1,85 @@
+packs:
+  - uid: mondoo-asset-inventory-gcp
+    name: Mondoo mondoo-asset-inventory-gcp Asset Inventory
+    tags:
+      mondoo.com/platform: gcp,cloud
+      mondoo.com/category: best-practices
+    docs:
+      desc: |
+        The Mondoo GCP Inventory query pack gathers information about GCP projects for asset inventory.
+    filters:
+      - asset.platform == "gcp"
+      - uid: mondoo-asset-inventory-gcp-project-info
```

Comment:
queries keyword is missing

Reply:
Wow...I am not sure how the pack is working at all. 🤔  Regardless, thanks finding that @chris-rock